aws/s3-cdn/s3-cdn.tf

```diff
@@ -0,0 +1,557 @@
+}
+
+resource "aws_cloudfront_distribution" "default" {
+  enabled             = var.create
+  is_ipv6_enabled     = var.is_ipv6_enabled
+  comment             = format("%s %s", var.desc_prefix, var.comment)
+  default_root_object = var.default_root_object
+  price_class         = var.price_class
+  depends_on          = [aws_s3_bucket.origin]
+
+  logging_config {
+    include_cookies = var.log_include_cookies
+    bucket          = module.logs.bucket_domain_name
+    prefix          = var.log_prefix
+  }
+
+  aliases = sort(
+      distinct(compact(concat(var.aliases, var.external_aliases))),
+    )
+  # aliases = var.aliases
+
+  origin {
+    domain_name = local.bucket_domain_name
+    origin_id   = local.module_prefix
+    origin_path = var.origin_path
+
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.default.cloudfront_access_identity_path
+    }
+  }
+
+  viewer_certificate {
+    acm_certificate_arn            = var.acm_certificate_arn
+    ssl_support_method             = "sni-only"
+    minimum_protocol_version       = var.minimum_protocol_version
+    cloudfront_default_certificate = var.acm_certificate_arn == "" ? true : false
+  }
+
+  default_cache_behavior {
+    allowed_methods  = var.allowed_methods
+    cached_methods   = var.cached_methods
+    target_origin_id = local.module_prefix
+    compress         = var.compress
+    trusted_signers  = var.trusted_signers
+
+    forwarded_values {
+      query_string = var.forward_query_string
+      headers      = var.forward_header_values
+
+      cookies {
+        forward = var.forward_cookies
+      }
+    }
+
+    viewer_protocol_policy = var.viewer_protocol_policy
+    default_ttl            = var.default_ttl
+    min_ttl                = var.min_ttl
+    max_ttl                = var.max_ttl
+
+    dynamic "lambda_function_association" {
+      for_each = var.lambda_function_association
+      content {
+        event_type   = lambda_function_association.value.event_type
+        include_body = lookup(lambda_function_association.value, "include_body", null)
+        lambda_arn   = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  restrictions {
+    geo_restriction {
+      restriction_type = var.geo_restriction_type
+      locations        = var.geo_restriction_locations
+    }
+  }
+
+  dynamic "custom_error_response" {
+    for_each = var.custom_error_response
+    content {
+      error_caching_min_ttl = lookup(custom_error_response.value, "error_caching_min_ttl", null)
+      error_code            = custom_error_response.value.error_code
+      response_code         = lookup(custom_error_response.value, "response_code", null)
+      response_page_path    = lookup(custom_error_response.value, "response_page_path", null)
+    }
+  }
+
+  web_acl_id          = var.web_acl_id
+  wait_for_deployment = var.wait_for_deployment
+
+  tags = local.tags
+}
+
+module "dns" {
```

Comment:
Are all resources/modules that support tags/comments have them associated?

Reply:
Yes, double checked and all supported resources have tags/comments/descriptions